src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -66,7 +67,7 @@ function ProductGallery({ title }: Props) {
       {next !== false && (
         <>
           <GatsbySeo linkTags={[{ rel: 'next', href: next.link }]} />
-          <a
+          <LinkButton
```

Comment:
IMO, it should be an anchor. Buttons don't handle opening new tabs, and anchor has a better semantic for the scenario since this "button" controls navigation.